apps/admin-x-activitypub/src/api/activitypub.test.ts

```diff
@@ -235,6 +278,43 @@ describe('ActivityPubAPI', function () {
 
             expect(actual).toEqual(expected);
         });
+
+        test('Returns an array when the items key is a single object', async function () {
+            const fakeFetch = Fetch({
+                'https://auth.api/': {
+                    response: JSONResponse({
+                        identities: [{
+                            token: 'fake-token'
+                        }]
+                    })
+                },
+                'https://activitypub.api/.ghost/activitypub/following/index': {
+                    response:
+                     JSONResponse({
+                         type: 'Collection',
+                         items: {
+                             type: 'Person'
+                         }
+                     })
+                }
+            });
+
+            const api = new ActivityPubAPI(
+                new URL('https://activitypub.api'),
+                new URL('https://auth.api'),
+                'index',
+                fakeFetch
+            );
+
+            const actual = await api.getFollowing();
+            const expected: Activity[] = [
```

Comment:
Is this the correct type here?

Reply:
Ah looked up the type, its an alias for `any` 😄 I'm assuming we have future plans to refactor this?

Reply:
Yes definitely! The main thing for these changes was to get the network out of the components so we can test